release-notes/opensearch.release-notes-2.6.0.md

```diff
@@ -5,13 +5,10 @@
 - Add index create block when all nodes have breached high disk watermark ([#5852](https://github.com/opensearch-project/OpenSearch/pull/5852))
 - Add cluster manager throttling stats in nodes/stats API ([#5790](https://github.com/opensearch-project/OpenSearch/pull/5790))
 - Add support for feature flags in opensearch.yml ([#4959](https://github.com/opensearch-project/OpenSearch/pull/4959))
-- Add query for initialized extensions ([#5658](https://github.com/opensearch-project/OpenSearch/pull/5658))
```

Comment:
These were also removed from the CHANGELOG from the 2.x section: https://github.com/opensearch-project/OpenSearch/blob/main/CHANGELOG.md
Can you please add it back if it's unreleased?

Reply:
Thanks for noticing it. Pushed the change!